openc3/lib/openc3/utilities/s3.rb

```diff
@@ -17,7 +17,9 @@
 # All changes Copyright 2022, OpenC3, Inc.
 # All Rights Reserved
 
-autoload(:Aws, 'openc3/utilities/s3_autoload.rb')
+module Aws
+  autoload(:S3, 'openc3/utilities/s3_autoload.rb')
+end
```

Comment:
Was this autoloading the entire AWS API? Wow, nice change!

Reply:
No, but it did autoload S3 on the first reference to Aws.  Now you have to actually reference S3.